js/widgets/temperament.js

```diff
@@ -1774,9 +1774,11 @@ function TemperamentWidget() {
         for (let i = 0; i < this.pitchNumber; i++) {
             const idx = newStack.length;
             if (
-                this.inTemperament === "equal" ||
+               (this.inTemperament === "equal" || this.inTemperament === "equal5" ||
```

Comment:
I forget how to do this in JS, but why not check to see that the first 5 letters are "equal" (begins with).

Reply:
ohh so you want me to reduce it to check if the string starts with equal must follow the if otherwise else